ci/kokoro/lib/gcloud.sh

```diff
@@ -30,38 +34,45 @@ if [[ -z "${GCLOUD_ARGS:-}" ]]; then
   )
 fi
 
+GCLOUD_BIN="/usr/local/google-cloud-sdk/bin/gcloud"
+if [ ! -x "${GCLOUD_BIN}" ]; then
+  # If the bin wasn't found at the above path, hope it's found in the path.
```

Comment:
A few too many "path"s perhaps?  Maybe `s/in the path/in ${PATH}/`?

Reply:
Fixed.